catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/SortedQueryMonitor.java

```diff
@@ -163,8 +163,7 @@ public void run() {
                         Exceptions.getFullMessage(e),
                         e);
 
-                processingDetails.add(new ProcessingDetailsImpl(sourceId,
-                        new Exception(Exceptions.getFullMessage(e))));
+                processingDetails.add(new ProcessingDetailsImpl(sourceId, e));
```

Comment:
Furthermore, note that we **will** get a useful message (without `Exceptions.getFullMessage`) because I'm recursively scanning `Exception.getCause()`.